gax/status.h

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google Inc.  All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_GAX_STATUS_H_
+#define GOOGLE_GAX_STATUS_H_
+
+#include <ostream>
+#include <string>
+
+namespace google {
+namespace gax {
+
+enum class StatusCode {
```

Comment:
Can we keep (or rewrite) the comment that links to the upstream source of truth for this enum?

Reply:
Good catch. I thought I had added it, but it looks like it's just in the commit message, which is not helpful to anyone perusing the code.